src/components/scenes/SettingsScene.tsx

```diff
@@ -105,17 +105,14 @@ export class SettingsSceneComponent extends React.Component<Props, State> {
     if (!this.props.supportsTouchId) return
 
     const biometryType = await getSupportedBiometryType()
+    if (Platform.OS === 'android') this.setState({ touchIdText: s.strings.settings_button_use_biometric })
```

Comment:
This is the opposite of what we talked about, which is to make iOS the special case. You could fix this by doing `if (Platform.OS !== 'ios') ...biometric`, or something like that. Any platform other than iOS (Desktop?) should just use "biometric"